datasafe-test-storages/src/main/java/de/adorsys/datasafe/teststorage/WithStorageProvider.java

```diff
@@ -281,6 +347,45 @@ private static void initS3() {
         log.info("Amazon napped URL:" + amazonMappedUrl);
     }
 
+    private static void initMultiS3(){
+        if(amazons3BucketCount == null){
+            amazonMappedUrlList = new String[0];
+            amazonMappedUrlList[0] = amazonMappedUrl;
+        }
+        amazonS3List = new ArrayList<>();
+        Integer s3BucketCount = Integer.parseInt(amazons3BucketCount);
+        amazonMappedUrlList = new String[s3BucketCount];
+
+        for(int i=1; i <= s3BucketCount; i++){
```

Comment:
Why i can't start from 0 ?
MultiS3 by itself expects multiple buckets available, so
valid list is:
bucket-0,bucket-1
but not
bucket, bucket-1

Reply:
updated